tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -188,20 +196,40 @@ else
 EOL
 fi
 
-kubetest2 kops "${KUBETEST2_ARGS[@]}" \
-  --up \
-  --kubernetes-version="${K8S_VERSION}" \
-  --create-args="${create_args[*]}" \
-  --test=clusterloader2 \
-  -- \
-  --provider="${CLOUD_PROVIDER}" \
-  --repo-root="${GOPATH}"/src/k8s.io/perf-tests \
-  --test-configs="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/load/config.yaml \
-  --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/load/overrides.yaml \
-  --extra-args="--experimental-prometheus-snapshot-to-report-dir=true" \
-  --kube-config="${HOME}/.kube/config"
-  # --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/experiments/enable_restart_count_check.yaml \
-  # --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/experiments/ignore_known_gce_container_restarts.yaml \
-  # --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/overrides/5000_nodes.yaml \
-  # --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/huge-service/config.yaml \
-  # --test-overrides="${GOPATH}"/src/k8s.io/perf-tests/clusterloader2/testing/access-tokens/config.yaml \
+CLUSTERLOADER2_ARGS=()
+if [[ -n "${KOPS_CL2_TEST_CONFIG}" ]]; then
+  CLUSTERLOADER2_ARGS+=("--test-configs=${KOPS_CL2_TEST_CONFIG}")
```

Comment:
@upodroid can we make this `CLUSTERLOADER2_ARGS+=("--test-configs=${GOPATH}/src/k8s.io/perf-tests/clusterloader2/${KOPS_CL2_TEST_CONFIG}")`?